src/app/video-courses/courses/course/date-highlight/date-highlight.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, HostBinding, Input, OnInit } from '@angular/core';
```

Comment:
directives and pipes should be in the shared with standalone folders 'directives' and 'pipes' respectively